packages/legacy/core/App/screens/PINExplainer.tsx

```diff
@@ -38,8 +42,8 @@ const PINExplainer: React.FC<PINExplainerProps> = ({ continueCreatePIN }) => {
   }
 
   return (
-    <ScrollView contentContainerStyle={{ flexGrow: 1 }}>
-      <View style={style.screenContainer}>
+    <SafeAreaView style={style.safeAreaView} edges={['bottom', 'left', 'right']}>
```

Comment:
Because there is a header on this screen, which handles the top safe area already. Including it here would just cause extra space below the header and above the content